app/uk/gov/hmrc/pensionschemereturn/transformations/nonsipp/MemberPaymentsTransformer.scala

```diff
@@ -61,7 +67,13 @@ class MemberPaymentsTransformer @Inject()(
           memberLumpSumReceived = memberDetails.memberLumpSumReceived.map(
             x => List(EtmpMemberLumpSumReceived(x.lumpSumAmount, x.designatedPensionAmount))
           ),
-          memberTransfersOut = memberDetails.transfersOut.map(transferOutTransformer.toEtmp)
+          memberTransfersOut = memberDetails.transfersOut.map(transferOutTransformer.toEtmp),
+          memberPensionSurrender = if (memberPayments.benefitsSurrenderedDetails.made) {
```

Comment:
do we still need this if (memberPayments.benefitsSurrenderedDetails.made) check here ? 
would it be better if we rely on the optional surrender list now as memberLumpSumReceived ? 

and If we still need that check, 

we may simplify here as below if you like :

`Option.when(memberPayments.benefitsSurrenderedDetails.made)(
            memberDetails.benefitsSurrendered.map(pensionSurrenderTransformer.toEtmp) match {
              case Some(surrender) => List(surrender)
              case None => Nil
            }
          )`


But either way is good

Reply:
I like it! will change